src/Components/SimpleModal/SimpleModal.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { StyleSheet, TouchableOpacity, View } from "react-native";
+import Modal, { ModalProps } from "react-native-modal";
+import getShadowStyles from "src/Themes/getShadowStyle";
+import { isWeb } from "src/Helpers/platform";
+import Colors from "src/Themes/Colors";
+
+const documentBody = document.getElementsByTagName("body")[0];
+
+interface IProps {
+  isVisible: ModalProps["isVisible"];
+  children: ModalProps["children"];
+  onBackdropPress?: ModalProps["onBackdropPress"];
+  onModalShow?: ModalProps["onModalShow"];
+}
+
+export class SimpleModal extends React.PureComponent<IProps> {
+  private el: Node;
+  constructor(props) {
+    super(props);
+    this.el = document.createElement("div");
+  }
+  public componentDidMount = () => {
+    if (!documentBody) return;
```

Comment:
This check here checks if `documentBody` is undefined or not. You return here because you know, that `documentBody` is undefined in NOT web environment, so we do not need to create elements.

It is not a good approach to do un-explicit checks, by the following reasons:

- It is not understandable why there is a check here
- You do not cover all the use-cases. For example, in some scenarios `documentBody` can be undefined even in web environment. For example, if code runs in a more secure environment, where `document.body` is not accessible. In this case your code will not event try to create a node, but will just fail silently. But it should fail not silently.